src/network/networkd-dhcp6.c

```diff
@@ -363,6 +363,29 @@ static int dhcp6_pd_address_handler(sd_netlink *rtnl, sd_netlink_message *m, Lin
         return 1;
 }
 
+static void log_dhcp6_pd_address(Link *link, const Address *address) {
+        char valid_buf[FORMAT_TIMESPAN_MAX], preferred_buf[FORMAT_TIMESPAN_MAX];
+        const char *valid_str = NULL, *preferred_str = NULL;
+        _cleanup_free_ char *buffer = NULL;
+        int log_level;
+
+        log_level = address_get(link, address, NULL) >= 0 ? LOG_DEBUG : LOG_INFO;
```

Comment:
Not too familiar with these functions - if this is in the hot/critical path, it might be worth to micro-optimize and return here if current log level < log_level, to avoid doing the formatting work just to have it thrown away. But only if this function is called in perf-critical path, otherwise it's fine as-is.

Reply:
Done.